src/main/java/com/google/sps/utilities/MockTutorSessionDatastore.java

```diff
@@ -65,4 +65,27 @@ public void addTutorSession(String tutorEmail, String studentEmail, TutorSession
         return student.getScheduledSessions();
     }
 
+    /**
+    * Adds the given rating to a tutor session and updates the tutor's overall rating.
```

Comment:
I ended up creating a different interface and a different mock for rating but I think it's easier to keep the rating functions inside the TutorSessionDatastoreService. We can just use this approach after we merge.